packages/styling/src/utilities/index.ts

```diff
@@ -15,3 +15,17 @@ export {
 export {
   getIconClassName
 } from './getIconClassName';
+
+export const HighContrastSelector = '@media screen and (-ms-high-contrast: active)';
+
+export const ScreenWidthMinSmall = 320;
```

Comment:
Put these constants in a different module?

Reply:
(And re-export them from the index)

Reply:
Yup good suggestion.